draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -602,8 +603,9 @@ Here, `eyJhbGci...emhlaUJhZzBZ` represents the SD-JWT and `eyJhb...dYALCGg` repr
 ## Mandatory digest computation of the revealed claim values by the Verifier
 
 ToDo: add text explaining mechanisms that should be adopted to ensure that
-  Verifiers validate the claim values received in HS-Disclosures JWT by calculating the
-  digests of those values and comparing them with the digests in the SD-JWT:
+
+Verifiers validate the claim values received in Disclosures by calculating the
+digests of those values and comparing them with the digests in the SD-JWT:
```

Comment:
This change breaks the sentence above. I suggest to **remove this section completely:** It comes from a time when the format did force Verifiers to hash the disclosures. Also, manipulation of disclosures is discussed below.

Reply:
agreed to remove.